pages/glossary/included-in-the-accessibility-tree.md

```diff
@@ -11,7 +11,7 @@ Elements included in the accessibility tree of platform specific accessibility A
 
 The general rules for when elements are included in the accessibility tree are defined in the [core accessibility API mappings](https://www.w3.org/TR/core-aam/). For native markup languages, such as HTML and SVG, additional rules for when elements are included in the accessibility tree can be found in the [HTML accessibility API mappings (working draft)](https://www.w3.org/TR/html-aam/) and the [SVG accessibility API mappings (working draft)](https://www.w3.org/TR/svg-aam/).
 
-For more details, see [examples of included in the accessibility tree][].
+For more details, see [examples of included in the accessibility tree](https://act-rules.github.io/pages/examples/included-in-the-accessibility-tree/).
+
+Elements with `aria-hidden="true"` should not be included in the accessibility tree. However, some browsers include [focusable](#focusable) elements with `aria-hidden="true"`in the accessibility tree. In such a browser, a user of assistive technology could still interact with the element by using sequential keyboard navigation, even though the element should not be included in the accessibility tree.
```

Comment:
My preference would be to Remove the "**Note**" word, leave the text as is and then on the bottom of that text to add a "**Note**" that would say that some browsers tend to expose focusable elements with `aria-hidden` but this behaviour is not consistent; therefore we consider them not exposed.

It's not a strong preference though, so it would be good to know what others think about it.

Reply:
I think it might be well in a "Background" section, like we have for [Block of repeated content](https://act-rules.github.io/glossary/#block-of-repeated-content) or [Instrument to achieve an objective](https://act-rules.github.io/glossary/#instrument-to-achieve-an-objective).